SpatialGDK/Source/SpatialGDK/Public/SpatialConstants.h

```diff
@@ -85,6 +85,7 @@ const Worker_PartitionId INVALID_PARTITION_ID = INVALID_ENTITY_ID;
 
 const Worker_ComponentId INVALID_COMPONENT_ID = 0;
 
+const Worker_ComponentId WELLKNOWNCOMPONENTSET_COMPONENT_ID = 50;
```

Comment:
Nit : Could you prefix that by SPATIALOS ? We have a fair amount of "well known" stuff now.

Reply:
Sure, done.